src/litserve/loops.py

```diff
@@ -399,63 +400,176 @@ class _BaseLoop(ABC):
       - unbatch
       - encode_response
 
+    To implement a custom loop, subclass this class and implement the `run` method. The `run` method should execute the
```

Comment:
docstring are in most cases RST formatted not MD

Reply:
I would say Markdown is easier to understand for most of the users and we are not using Sphinx here.